src/Peachpie.CodeAnalysis/Semantics/PhpOperationVisitor.cs

```diff
@@ -130,7 +130,11 @@ public virtual void VisitList(BoundListEx x)
 
         public virtual void VisitFieldRef(BoundFieldRef x)
         {
-            VisitTypeRef(x.ParentType);
+            if (x.ParentType != null)
```

Comment:
Ah.. I see.. thanks for the explanation. Moved in d68f95462d7da012483a47ab2bf01842afaa6079.